components/Button/src/index.scss

```diff
@@ -178,3 +178,15 @@ a.denhaag-button {
 .denhaag-button--icon-only .denhaag-button__icon {
   height: var(--denhaag-button-icon-only-icon-height);
 }
+
+.utrecht-heading-1 + .denhaag-button,
```

Comment:
This could be a lot shorter:
`[class^='utrecht-heading'] + .denhaag-button, [class*=' utrecht-heading'] + .denhaag-button`